server/manager/src/main/java/org/apache/accumulo/manager/merge/FindMergeableRangeTask.java

```diff
@@ -132,18 +133,17 @@ void submit(MergeableRange range, FateInstanceType type, Entry<TableId,String> t
     String endRowStr = StringUtils.defaultIfBlank(endRow.toString(), "+inf");
     log.debug("FindMergeableRangeTask: Creating merge op: {} from startRow: {} to endRow: {}",
         tableId, startRowStr, endRowStr);
-    var fateId = manager.fate(type).startTransaction();
-    String goalMessage = TableOperation.MERGE + " Merge table " + tableName + "(" + tableId
-        + ") splits from " + startRowStr + " to " + endRowStr;
+    var fateKey = FateKey.forMerge(new KeyExtent(tableId, range.endRow, range.startRow));
 
-    manager.fate(type).seedTransaction(FateOperation.SYSTEM_MERGE, fateId,
+    manager.fate(type).seedTransaction(FateOperation.SYSTEM_MERGE, fateKey,
```

Comment:
This will wait for the data to be written to the walog of the fate table for each operation seeded.  W/ the changes in #5160 this is another place where waiting on the walog per operation can be avoided.